packages/design-system/src/components/Table/Table.jsx

```diff
@@ -43,7 +45,7 @@ export class Table extends React.PureComponent {
       if (
         props.scrollable &&
         Array.isArray(props.children) &&
-        !props.children.some((child) => child.type === TableCaption)
+        !props.children.some((child) => child.type.displayName === 'TableCaption')
```

Comment:
Can we reuse `isTableCaption` here?

Reply:
Yes, using the default prop `isTableCaption` will reduce the bundle size.